src/microbe_stage/MicrobeAISystem.cs

```diff
@@ -10,9 +10,12 @@ public class MicrobeAISystem
 
     private readonly Node worldRoot;
 
-    public MicrobeAISystem(Node worldRoot)
+    private readonly CompoundCloudSystem clouds;
```

Comment:
Could you add a cautionary comment here as the AI is run in threaded mode, the AI may *not* call any state changing methods on the cloud system (only data reading methods are allowed).